api/psalm.xml

```diff
@@ -1,8 +1,11 @@
 <?xml version="1.0"?>
+<!--suppress XmlDefaultAttributeValue for findUnusedCode -->
 <psalm
     errorLevel="6"
     resolveFromConfigFile="true"
     cacheDirectory="./.cache"
+    findUnusedBaselineEntry="true"
+    findUnusedCode="false"
```

Comment:
Is `findUnusedCode` disabled by purpose? Or would this be nice to enable?

Reply:
https://github.com/ecamp/ecamp3/pull/3193/commits/e0cca66095588f84f549b9ddd40f61bc7fc91b6a